### PR TITLE
Adds age coupling kernel and some cleaning

### PR DESCRIPTION
**Pull request by tozanski**

### CODE REVIEW

File: src/simparams.jl [hunk 1]
```diff
@@ -140,7 +140,11 @@ function make_params(
   spreading_x0::Real=1,
   spreading_truncation::Real=Inf,
 
-  british_strain_multiplier::Real=1.70
+  british_strain_multiplier::Real=1.70,
+  delta_strain_multiplier::Real=1.7*1.5,
```
Comment (eMaerthin) [inline src/simparams.jl:144]: why you need 2 places: here and `strains.jl` to specify multipliers?

**Review by eMaerthin — COMMENTED** [review]
why it is still WIP? Not tested?
overall changes sound good, but I wish there will be some tests added soon.

File: src/simparams.jl (hunk 1 above)
Comment (tozanski) [reply src/simparams.jl:144]: These are just default values. Perhaps we should just make them constant. What value do we use for the british variant and what for the delta variant?

Comment (eMaerthin) [reply src/simparams.jl:144]: yes, constants you gave are correct

**Pull request merged**